diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -146,6 +147,16 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    private fun isReferenceExpressionWithSameName(node: ASTNode, kdocTags: Collection<KDocTag>?): Boolean {
+        if (node.isSingleLineGetterOrSetter()) {
+            return false
+        }
+        val lastDotQualifiedExpression = node.findChildByType(DOT_QUALIFIED_EXPRESSION)?.psi?.let { (it as KtDotQualifiedExpression).lastChild?.text?.substringBefore('(') }
+        val funName = (node.psi as KtFunction).name
+        return funName == lastDotQualifiedExpression
+    }
+
+    @Suppress("WRONG_NEWLINES")
```

Comment:
Why did you suppress it?)

Reply:
The Diktat merge 2 lines into one, and then breaks down on LONG_LINE

Reply:
Then please create an issue with a reproducer